tests/with_ray/full_wlm/test_ray_pbs_batch.py

```diff
@@ -0,0 +1,74 @@
+import logging
+import sys
+import time
+from os import environ
+from shutil import which
+
+import pytest
+
+from smartsim import Experiment
+from smartsim.exp.ray import RayCluster
+
+"""Test Ray cluster Slurm launch and shutdown.
+"""
+
+# retrieved from pytest fixtures
+if pytest.test_launcher not in pytest.wlm_options:
+    pytestmark = pytest.mark.skip(reason="Not testing WLM integrations")
+
+
+environ["OMP_NUM_THREADS"] = "1"
+try:
+    import ray
```

Comment:
The ``sys.module`` approach doesn't work in some cases for me.

see https://github.com/Spartee/SmartSim/blob/c00d8b301ea9dfb99dda4faba1dd83c3212ee739/tests/backends/test_onnx.py#L9

Reply:
OK, updated on all Ray tests.